### ISSUE TITLE
Target click size for AAA

### ISSUE THREAD

**Issue by colinreedmiller** [vmware/clarity#3333]
A11y have suggested that a theme with generous spacing and larger targets should be added to Clarity.

Target size is AAA – I can’t make you do it, but I really really want you to
Success Criterion 2.5.5 Target Size

(Level AAA)

The size of the target for pointer inputs is at least 44 by 44 CSS pixels except when:
Equivalent
The target is available through an equivalent link or control on the same page that is at least 44 by 44 CSS pixels;

Inline
The target is in a sentence or block of text;

User Agent Control
The size of the target is determined by the user agent and is not modified by the author;

Essential
A particular presentation of the target is essential to the information being conveyed.

**Comment by mathisscott** [comment]
This is dependent on theming improvements we are planning for 3.0.

**Comment by colinreedmiller** [comment]
Meeting this requirement is necessary to sell in the EU public sector.  It must be implemented in our PRODUCTS before 9/23/20, which means Clarity really needs to have it by EOY Calendar 19
 
Success Criterion 1.4.12 Text Spacing

(Level AA)

In content implemented using markup languages that support the following text style properties, no loss of content or functionality occurs by setting all of the following and by changing no other style property:
·       Line height (line spacing) to at least 1.5 times the font size;

·       Spacing following paragraphs to at least 2 times the font size;

·       Letter spacing (tracking) to at least 0.12 times the font size;

·       Word spacing to at least 0.16 times the font size.

·        

Target size is AAA – I can’t make you do it, but I really really want you to
Success Criterion 2.5.5 Target Size

(Level AAA)

The size of the target for pointer inputs is at least 44 by 44 CSS pixels except when:
Equivalent
The target is available through an equivalent link or control on the same page that is at least 44 by 44 CSS pixels;

Inline
The target is in a sentence or block of text;

User Agent Control
The size of the target is determined by the user agent and is not modified by the author;

Essential
A particular presentation of the target is essential to the information being conveyed.

**Comment by colinreedmiller** [comment]
(from Sheri)

Our recommendation is to provide the default compact solution that we are currently using with the option for the user to expand to the more spacious version upon request.  Preferably this would be integrated as a permanent login profile item, but I’ll take what I can get out of the gate.

**Comment by gnomeontherun** [comment]
Dev is blocked but design can proceed. Will also need to update design assets.

**Comment by gnomeontherun** [comment]
I'm going to close this for now until we have stronger guidance on what to do here and if we want to tackle it, as its not just a matter of adding padding to make a larger theme.
